### DISCUSSION TITLE
Package doesn't have a strong name

### DISCUSSION THREAD

**Discussion by gheorghe-stefan** [DotSpatial/DotSpatial#1460] [General]
Hello,
I encounter the following error when using the latest DotSpatial version in my project:
`error CS8002: Referenced assembly 'DotSpatial.Projections, Version=4.0.656.0, Culture=neutral, PublicKeyToken=null' does not have a strong name.`
Is there a chance to make a new version with it signed?

Thanks in advance,
Stefan G

**Comment by jany-tenaj** [comment]
We're not going to do this.

**Reply by gheorghe-stefan** [reply]
Thanks for your reply Jany; what would be an alternative then, in order to use your package?

**Reply by jany-tenaj** [reply]
Get rid of whatever forces you to have strong names?

**Reply by gheorghe-stefan** [reply]
good point, thanks Jany!
